dev/Repeater/InteractionTests/RepeaterTests.cs

```diff
@@ -22,6 +27,64 @@ namespace Windows.UI.Xaml.Tests.MUXControls.InteractionTests
     [TestClass]
     public class RepeaterTests
     {
-        // Placeholder for interaction tests
+        [ClassInitialize]
+        [TestProperty("RunAs", "User")]
+        [TestProperty("Classification", "Integration")]
+        [TestProperty("TestPass:IncludeOnlyOn", "Desktop")]
+        public static void ClassInitialize(TestContext testContext)
+        {
+            TestEnvironment.Initialize(testContext);
+        }
+
+        public void TestCleanup()
+        {
+            TestCleanupHelper.Cleanup();
+        }
+
+        [TestMethod]
+        public void FocusedItemGetsRecycledUponCollectionReset()
+        {
+            Button navButton = new Button(FindElement.ByName("ItemsRepeater Tests"));
+            navButton.InvokeAndWait();
+
+            Wait.ForMilliseconds(1000);
+
+            navButton = new Button(FindElement.ByName("Collection Changes Demo"));
+            navButton.InvokeAndWait();
+
+            Wait.ForMilliseconds(1000);
+
+
+            Random r = new Random();
+            int index = r.Next(1, 10);
+            index = 2;
```

Comment:
>index = 2; [](start = 12, length = 10)

doesn't this defeat the purpose of your random item grabber?

Reply:
Oh I forgot to remove that 😅
Removed it, thanks for pointing that out.